gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -327,11 +327,13 @@ TFLiteCore::invoke (const GstTensorMemory * input, GstTensorMemory * output)
   }
 
   for (int i = 0; i < outputTensorMeta.num_tensors; i++) {
+    void * out;
     if (outputTensorMeta.info[i].type == _NNS_FLOAT32) {
-      output[i].data = interpreter->typed_output_tensor < float >(i);
+      out = interpreter->typed_output_tensor < float >(i);
     } else if (outputTensorMeta.info[i].type == _NNS_UINT8) {
-      output[i].data = interpreter->typed_output_tensor < uint8_t > (i);
+      out = interpreter->typed_output_tensor < uint8_t > (i);
```

Comment:
What if the type is  INT32 or INT16?
At least, you need to do validation check here. (explicit error for not supported types)

Reply:
it's updated for the various supported types.

Reply:
We don't need to compare the type actually.

Reply:
@jaeyun-jung I agree with your suggestion. It looks better!